.travis.yml

```diff
@@ -138,6 +138,7 @@ jobs:
         - travis_wait make test-cmd-devfile-watch
         - travis_wait make test-cmd-devfile-delete
         - travis_wait make test-cmd-devfile-registry
+        - travis_wait make test-cmd-devfile-test
```

Comment:
please update the name of the test to `test`
/hold

Reply:
I don't quite understand this comment.

Reply:
I meant updating the name of the job like https://github.com/openshift/odo/pull/3400/files#diff-354f30a63fb0907d4ad57269548329e3R161 below

Reply:
updated